app/src/main/java/com/work/restaurant/data/source/local/bookmark/BookmarkLocalDataSourceImpl.kt

```diff
@@ -0,0 +1,75 @@
+package com.work.restaurant.data.source.local.bookmark
+
+import com.work.restaurant.network.room.database.BookmarkDatabase
+import com.work.restaurant.network.room.entity.BookmarkEntity
+import com.work.restaurant.util.AppExecutors
+
+class BookmarkLocalDataSourceImpl(
+    private val appExecutors: AppExecutors,
+    private val bookmarkDatabase: BookmarkDatabase
+) : BookmarkLocalDataSource {
+
+    override fun addBookmark(
+        bookmarkEntity: BookmarkEntity,
+        callback: BookmarkLocalDataSourceCallback.AddBookmarkCallback
+    ) {
+        appExecutors.diskIO.execute {
+
+            val addBookmark = bookmarkDatabase.bookmarkDao().addBookmark(bookmarkEntity)
+            if (addBookmark >= 1) {
+                callback.onSuccess("successAdd")
```

Comment:
```suggestion
                callback.onSuccess(true)    혹은    callback.onSuccess()
```
어떠한 결과 메시지를 받을것이 아닌데 String 타입으로 결과를 받는것은 비효율적으로 보입니다.
String 자체가 하나의  객체여서 훨씬 load가 있거든요

Reply:
17db4cb 수정하였습니다.